tests/unit/backend/wmg/fixtures/test_snapshot.py

```diff
@@ -100,6 +100,15 @@ def exclude_random_coords_75pct(_) -> bool:
     return random() > 0.75
 
 
+def exclude_dev_stage_and_ethnicity_for_secondary_filter_test(coord) -> bool:
+    dev_stages_to_exclude = ("development_stage_ontology_term_id_1", "development_stage_ontology_term_id_2")
+    ethnicity_terms_to_exclude = ("ethnicity_ontology_term_id_1", "ethnicity_ontology_term_id_2")
+    if coord.development_stage_ontology_term_id in dev_stages_to_exclude:
```

Comment:
This and the following line could be put together with `and` but then the linter gets mad